src/engine.tsx

```diff
@@ -278,7 +278,9 @@ const Engine: {
       let offlineReputation = 0;
       const offlineHackingIncome =
         (Player.moneySourceA.hacking / Player.playtimeSinceLastAug) * timeOffline * CONSTANTS.OfflineHackingIncome;
-      Player.gainMoney(offlineHackingIncome, "hacking");
+      if (Number.isFinite(offlineHackingIncome)) {
+        Player.gainMoney(offlineHackingIncome, "hacking");
```

Comment:
I'd prefer the same fix here as elsewhere, in case someone decides to start using offlineHackingIncome later and subtly reintroduces the bug.